src/utils/hooks/useFetchUsersForInstitution.ts

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import Axios from 'axios';
+
+import { setNotification } from '../../redux/actions/notificationActions';
+import { NotificationVariant } from '../../types/notification.types';
+import { UserAdmin } from '../../types/user.types';
+import { getUsersForInstitution } from '../../api/roleApi';
+
+const useFetchUsersForInstitution = (institution: string): [UserAdmin[] | undefined, boolean] => {
+  const dispatch = useDispatch();
+  const [users, setUsers] = useState<UserAdmin[] | undefined>();
```

Comment:
typen  her heter `UserAdmin`, men det kan vel være annet enn admins også?
feks `Curator` eller `Publisher`

Reply:
Ja, endret ikke navn på denne fra hva det var tidligere for jeg kom ikke på noe bedre. Har du forslag? Føler det blir litt feil å kalle typen kun User også?

Reply:
ja, enig. InstitutionUser da? denne typen brukes sikkert bare i kontekst av customer institution?

Reply:
Ja, kan bruke det 👍